### PR TITLE
adding ignores

### PR DESCRIPTION
**Pull request by arschles**
for vscode config dir and binaries

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,6 @@ cmd/proxy/proxy
 cmd/olympus/olympus
 test-keys.json
 tmp
+.vs
```
Comment (robjloranger) [inline .gitignore]: has it changed? it used to be `.vscode`

Comment (robjloranger) [reply .gitignore]: oh weird, I thought it was the workspace settings folder

**Comment by arschles** [comment]
@robjloranger hrm, maybe it's coming from somewhere else? [this](https://stackoverflow.com/questions/31878901/should-i-add-the-visual-studio-2015-vs-folder-to-source-control) and [this](https://social.msdn.microsoft.com/Forums/en-US/1ea61876-d755-40e5-996f-0159fea46b10/vs-folder) indicate that it's a VS 2015 config dir.

i have no idea

anyway, are you open to adding while I figure out wtf is going on with my vscode install?

**Comment by robjloranger** [comment]
Sure

**Pull request merged**